### ISSUE TITLE
Jikkyō Oshaberi Parodius - random lockups on level transitions/load points.

### ISSUE THREAD

**Issue by Missingno255** [MiSTer-devel/PSX_MiSTer#263]
Jikkyō Oshaberi Parodius - or by English translated name, Chatting Parodius Live, has random lockup problems on the load points between stages. First encounter was after stage 2 (the high school), and then stages 4 and 5. Possibly reproducible, but it seems to be random and may not happen at all on a first run though of the game.

Name: Jikkyou Oshaberi Parodius - Forever with Me (Japan).bin
Size: 236070240 bytes (225 MiB)
CRC32: 6818A16D
Taken from Archive.org Redump collection.

Core version: PSX_20230503.rbf

**Comment by Kuba-J** [comment]
Did you play on the default settings? No speed hack , turbo , etc. ?

**Comment by Missingno255** [comment]
> Did you play on the default settings? No cd speed hack , turbo , etc. ?

AFAIK I didn't touch any of those settings, as I don't know what they do.

**Comment by RobertPeip** [comment]
Very reproducible and may not happen at all?

If you have a chance, please create a savestate at a spot where it happens often. Otherwise i see no chance to even look at that.

**Comment by Missingno255** [comment]
> Very reproducible and may not happen at all?
> 
> If you have a chance, please create a savestate at a spot where it happens often. Otherwise i see no chance to even look at that.

Yeah, I typoed, meant to say reproduceable, not "very reproduceable". I'll see if I can get a decent save state, I'm not very good at this game.

**Comment by Missingno255** [comment]
I just realized that I was on an unstable MiSTer main build at the time. I found that the "problem" wasn't happening at all, after 1 1/2 runs through the game, and past update may have put me back on stable, and probably didn't realize it, until now. So my apologies for the confusion and looking goofy. I shouldn't be reporting stuff breaking from an unstable main (used one to try the N64 core) anyways.
